src/web/components/SignInGate/SignInGateSelector.tsx

```diff
@@ -73,24 +79,28 @@ const tests: ReadonlyArray<ABTest> = [
 	signInGateMainVariant,
 	signInGateMainControl,
 	signInGateCopyOpt,
+	signInGateMandoryTest,
 ];
 
 const testVariantToGateMapping: GateTestMap = {
 	'main-control-3': gateMainControl,
-	'main-variant-3': gateMainVariant,
-	'copy-opt-control': gateMainVariant,
+	'main-variant-3': gateMainVariant(canShow),
+	'copy-opt-control': gateMainVariant(canShow),
 	'copy-opt-variant-1': gateCopyOptVar1,
 	'copy-opt-variant-2': gateCopyOptVar2,
 	'copy-opt-variant-3': gateCopyOptVar3,
 	'copy-opt-variant-4': gateCopyOptVar4,
 	'copy-opt-variant-5': gateCopyOptVar5,
 	'copy-opt-variant-6': gateCopyOptVar6,
+	'mandatory-gate-control': gateMainVariant(canShowMandatoryGate),
+	'mandatory-gate-variant': gateMandatoryVariant,
```

Comment:
I'm not sure entirely what's happening here with the `gateMainVariant` method?

Reply:
canShow is passed as a parameter to mainVariant because we need to have different canShow criteria for different tests

Reply:
Ah I see, that makes sense, just a note for myself: the `mandatory-gate-control` is the same design as the main variant test, but with different `canShow` rules, therefore the `gateMainVariant` method is used.